Packs/Workday/Integrations/WorkdayIAM/WorkdayIAM.yml

```diff
@@ -0,0 +1,67 @@
+category: IT Services
+commonfields:
+  id: WorkdayIAM
+  version: -1
+configuration:
+- display: Username
+  name: credentials
+  required: false
+  type: 9
+- display: Workday Report URL
+  name: report_url
+  required: true
+  type: 0
+- additionalinfo: Import schedule to get events
+  display: Fetch Events Frequency In minutes
+  name: fetch_events_time_minutes
+  required: false
+  type: 0
+- additionalinfo: Maximum number of events to fetch at a time
+  defaultvalue: '50'
+  display: Fetch Limit (Recommended less than 200)
+  name: max_fetch
+  required: false
+  type: 0
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- defaultvalue: '1'
+  display: Incidents Fetch Interval
+  name: incidentFetchInterval
+  required: false
+  type: 19
+- defaultvalue: IAM Sync User - Workday
+  display: Mapper (incoming)
+  hidden: false
+  name: mapper_in
+  required: false
+  type: 0
+description: Use the Workday IAM Integration as part of the IAM premium pack.
+display: Workday IAM
+name: WorkdayIAM
+script:
+  dockerimage: demisto/python3:3.8.6.12176
+  feed: false
+  isfetch: true
+  longRunning: false
+  longRunningPort: false
+  runonce: false
+  script: '-'
+  subtype: python3
+  type: python
+tests:
```

Comment:
add 
defaultClassifierId and defaultMapperIn
check the checkboxes to allow mapper in